package.json

```diff
@@ -85,8 +85,10 @@
         "test": "NODE_ICU_DATA=node_modules/full-icu BABEL_ENV=test NODE_ENV=test yarn jest -c scripts/jest/jest.config.js",
         "test:e2e": "BROWSERSLIST_ENV=test npm-run-all -p -r start:examples cy:run",
         "test:e2e:open": "BROWSERSLIST_ENV=test npm-run-all -p -r start:examples cy:open",
-        "test:images": "yarn test -c scripts/jest/jest.images.config.js --maxWorkers=4",
-        "test:visuals": "start-server-and-test start:storybook:ci http-get://localhost:6061 test:images",
+        "test:images": "NODE_ICU_DATA=node_modules/full-icu BABEL_ENV=test NODE_ENV=test yarn jest -c scripts/jest/jest.images.config.js --maxWorkers=4",
+        "test:images:snapshots": "NODE_ICU_DATA=node_modules/full-icu BABEL_ENV=test NODE_ENV=test yarn jest -c scripts/jest/jest.images.config.js -u --maxWorkers=4",
```

Comment:
this works though `"NODE_ICU_DATA=node_modules/full-icu BABEL_ENV=test NODE_ENV=test yarn jest -c scripts/jest/jest.images.config.js -u --maxWorkers=4 ./src/something/test.js"`

Reply:
I can do:
`yarn test:images -u src/components/avatar/__tests__/Avatar.stories.test.tsx`

using the new script `"test:images": "NODE_ICU_DATA=node_modules/full-icu BABEL_ENV=test NODE_ENV=test yarn jest -c scripts/jest/jest.images.config.js --maxWorkers=4",`

but I can't get `yarn test:visuals` or `yarn test:visuals:snapshots` to work for provided paths

Reply:
I thought the storybook test wont run unless you use `yarn test:visuals` if you do `yarn test:images` they fail

Reply:
Yeah the test won't succeed but I meant like I can run it and it'll take the path argument correctly.

Do you have an example of how we can use `yarn test:visual:snapshots` with the manual file path?

I'm trying `yarn test:visual:snapshots src/components/avatar/__tests__/Avatar.stories.test.tsx` but it's not working for me.

Reply:
you gotta add the file path at the end of `test:images:snapshots`  from visual  it doesnt trickle down

Reply:
So the developer would need to update the `test:images:snapshots` script in `package.json` to run `yarn test:visual:snapshots` as intended? That might seem unintuitive for some users.

If that (the above) was the intended flow, couldn't the developer add the file path to the existing `test:images` script and then run `yarn test:visuals` and then add the `-u` as needed?

From our Storybook doc, our recommended approach is to run `yarn start:storybook` and then the developer can run single tests with `yarn test:images $PATH_TO_TEST`. I think this might still be recommended as the developer won't need to start the server for each test.

Reply:
It doesn't really matter which way its written, it's really there as a helper command not as part of the process. I would prefer if we could do something like this `yarn test:visuals:snapshots $filepath` but that doesnt work or something like this `yarn test:visuals -u $filepath` but this doesnt work either. 

We can still keep the recommended approach but the other commands are really for us which can be explained as other options for different reasons.  You can still do `yarn start:storybook` with `yarn test:images $filepath` which would remain the recommended approach